cypress/integration/Dictionary/createDictionary.feature

```diff
@@ -0,0 +1,110 @@
+Feature: Dictionary Management
+
+  Scenario: A user to create a new dictionary
+    Given the user is logged in
+    And is on the user collections page
+    Then the user clicks on Create Dictionary Icon
+     Then the user should be able to create new dictionary
```

Comment:
NB I know this style is a little difficult to write in because the commands are actually aligned by their last letter rather than by leading space. The idea, though, is that this way, the relevant text (i.e. the parts beginning "the user") all start at the same column, which makes it a bit visually easier to see.

```suggestion
    Given the user is on the user collections page
     Then the user clicks on Create Dictionary icon
     Then the user should sent to the Create Dictionary page
```